tools/scripts/release_package.py

```diff
@@ -91,6 +91,34 @@ def _update_android_versions(version: str, github_token: str):
 
     uv._update_android_version(version)
 
+def _update_native_sdk_versions(version: str, arg_ios_version:str, arg_android_version: str, native_sdk_versions_path: str):
+    ios_version = arg_ios_version or _get_current_ios_version()
+    android_version = arg_android_version or _get_current_android_version()
+
+    if not ios_version or not android_version:
+        print(f"Error: Some native versions could not be retrieved, aborting creation of {dest_native_sdk_versions}")
+
+    if not os.path.exists(native_sdk_versions_path):
+        lines = [
+        "| Unity SDK | iOS SDK | Android SDK |\n",
+        "|-----------|---------|-------------|\n"
+    ]
+    else:
+        with open(native_sdk_versions_path, "r") as f:
+            lines = f.readlines()
+    
+    new_row = f"| {version} | {ios_version} | {android_version} |\n"
+    if len(lines) > 2 and lines[2] == new_row:
+        print(f"Entry for version {version} already present in {native_sdk_versions_path}")
```

Comment:
It might be worth doing a more full search here checking if the Unity version we're trying to ship already exists in the table. This only checks that the first line of the row is exactly matches the combination we're trying to ship.

Reply:
I agree. I've improved the existing entry checks here: [9149930](https://github.com/DataDog/dd-sdk-unity/pull/131/commits/9149930b420b5e5f4c38004d7dc5e18a71018114)